### ISSUE TITLE
Issues following example code

### ISSUE THREAD

**Issue by dazinator** [dotnet/EntityFramework.Docs#4065]
I was testing out `FromSql() ` by following these docs.

I encountered an error when executing the query:

```
System.InvalidCastException : The SqlParameterCollection only accepts non-null SqlParameter type objects, not SqlParameter objects.

```

This was an odd error message. It turns out, when following the sample code and creating a new `SqlParameter` object, there is actually a choice between `Microsoft.Data.SqlClient.SqlParameter` and `System.Data.SqlClient.SqlParameter`.

![image](https://user-images.githubusercontent.com/3176632/193090461-8ea10699-2550-4255-a978-a333dc90accb.png)

If you use the wrong one, you'll get this error. I happened to have the wrong namespace. These are two commonly used namespaces, perhaps a mention of this as a warning would be helpful.

#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 98b01ef5-24a5-5f59-30c9-af264f739815
* Version Independent ID: 98b01ef5-24a5-5f59-30c9-af264f739815
* Content: [SQL Queries - EF Core](https://learn.microsoft.com/en-us/ef/core/querying/sql-queries)
* Content Source: [entity-framework/core/querying/sql-queries.md](https://github.com/dotnet/EntityFramework.Docs/blob/main/entity-framework/core/querying/sql-queries.md)
* Product: **entity-framework**
* Technology: **entity-framework-core**
* GitHub Login: @smitpatel
* Microsoft Alias: **avickers**

**Comment by roji** [comment]
@dazinator that's indeed something that happens when users reference both versions of SqlClient, which is quite a rare case; I'm not sure it warrants a note in these docs. In addition, the docs are general for all databases, so we shouldn't add too much content that's specific to SQL Server here.

**Comment by ErikEJ** [comment]
I have fixed that error message in SqlClient 5.0!

**Comment by dazinator** [comment]
Ok, will close if you think not worth a docs update
